pyinjective/async_client.py

```diff
@@ -991,3 +991,14 @@ async def get_binary_options_market(self, market_id: str):
     async def get_account_portfolio(self, account_address: str):
         req = portfolio_rpc_pb.AccountPortfolioRequest(account_address=account_address)
         return await self.stubPortfolio.AccountPortfolio(req)
+
+    async def stream_account_portfolio(
+            self, account_address: str = None, subaccount_id: str = None, portfolio_type: str = None
+    ):
```

Comment:
Let's keep the method declarations in `async_client` consistent—we can list the required arguments like `account_address`  but for optional args let's use kwargs like in the other methods.

Reply:
I think we can also omit default arguments so that the user gets a missing argument error before they actually query the Indexer

Reply:
Thank you Hans, I've updated the code to use kwargs and tested passing a with subaccount Id. All good!
```bash
(sdk-python) ➜  sdk-python git:(feat/stream-account-portfolio) ✗ python3 examples/exchange_client/portfolio_rpc/2_StreamAccountPortfolio.py

Account portfolio Update:

type: "total_balances"
denom: "peggy0xdAC17F958D2ee523a2206206994597C13D831ec7"
amount: "1390858774431.917939947908988149"
subaccount_id: "0xb7434ee57ef509e5e12ac0c3ce8b7ebd6a1adb42000000000000000000000000"

Account portfolio Update:

type: "total_balances"
denom: "peggy0xdAC17F958D2ee523a2206206994597C13D831ec7"
amount: "1391526742431.917939947908988149"
subaccount_id: "0xb7434ee57ef509e5e12ac0c3ce8b7ebd6a1adb42000000000000000000000000"